lib/screens/profile/profile_screen.dart

```diff
@@ -0,0 +1,222 @@
+import 'package:flutter/material.dart';
+import 'package:openflutterecommerce/config/theme.dart';
+import 'package:openflutterecommerce/widgets/widgets.dart';
+
+class ProfileScreen extends StatefulWidget {
+  ProfileScreen({Key key}) : super(key: key);
+
+  @override
+  _ProfileScreenState createState() => _ProfileScreenState();
+}
+
+class _ProfileScreenState extends State<ProfileScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: OpenFlutterScaffold(
+        body: ProfileWrapper(),
+        bottomMenuIndex: 4,
+      ),
+    );
+  }
+}
+
+class ProfileWrapper extends StatefulWidget {
+  @override
+  _ProfileWrapperState createState() => _ProfileWrapperState();
+}
+
+class _ProfileWrapperState extends State<ProfileWrapper> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          elevation: 0.0,
+          actions: <Widget>[
+            IconButton(
+              onPressed: () {},
+              icon: Icon(Icons.search),
+            ),
+          ],
+          automaticallyImplyLeading: false,
+        ),
+        body: ListView(
+          
+          children: <Widget>[
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                Padding(
+                  padding: const EdgeInsets.only(top: 40.0,left: 10.0),
+                  child: Container(
+                    child: Text(
+                      "My profile",
+                      style: TextStyle(
+                          fontWeight: FontWeight.bold, fontSize: 34.0),
+                    )
+                  ),
+                ),
+                SizedBox(
+                  height: 5.0,
+                ),
+                ListTile(
```

Comment:
could you create a _buildItem(with the properties...)? a lot of code is duplicated